documentation/asciidoc/topics/con_backups_automatic_offline.adoc

```diff
@@ -0,0 +1,21 @@
+[id='backups_automatic_offline-{context}']
+= Performance Considerations with Taking Backup Locations Offline
+Backup locations can automatically go offline when remote sites become
+unavailable. This prevents nodes from attempting to replicate data to offline
+backup locations, which can have a performance impact on your cluster because it
+results in error.
+
+You can configure how long to wait before backup locations go offline. A good
+rule of thumb is one or two minutes. However, you should test different wait
+periods and evaluate their performance impacts to determine the correct value
+for your deployment.
+
+For instance when {openshiftshort} terminates the site master pod, that backup
+location becomes unavailable for a short period of time until {ispn_operator}
+elects a new site master.
+
+When this happens, if the minimum wait time is not long enough then the backup
+locations will go offline and come back online too frequently.
```

Comment:
I would stop the sentence in `go offline`. A said, to bring the site back online, user intervention is required and the state transfer is needed to keep the data sync.